src/pubtools/_marketplacesvm/tasks/community_push/command.py

```diff
@@ -30,7 +30,7 @@ class CommunityVMPush(MarketplacesVMPush, AwsRHSMClientService):
     """Upload an AMI to S3 and update RHSM."""
 
     _REQUEST_THREADS = int(os.environ.get("COMMUNITY_PUSH_REQUEST_THREADS", "5"))
-    _PROCESS_THREADS = int(os.environ.get("COMMUNITY_PUSH_PROCESS_THREADS", "10"))
```

Comment:
I am sorry but this will not help much.
The main issue is there is relation between threadpool.
For each thread created for `_REQUEST_THREADS` one would create threadpool with size _PROCESS_THREADS.
But they would have the same name `"pubtools-marketplacesvm-community-push-regions"`
It even complicates reading logs as well

Which s different situation for env variables for marketplace MARKETPLACESVM_PUSH_REQUEST_THREADS, MARKETPLACESVM_PUSH_PROCESS_THREADS

Reply:
Would it help if we try to rename the pools somehow?

Even the pools being related I believe reducing from 10 to 2 can help a bit as it won't create 50 threads, no?

Reply:
We use the env variable MARKETPLACESVM_PUSH_REQUEST_THREADS for 3p to set max limit how many threads can upload images in parallel.

For 1p we calculate it as a  
```
max_workers_a=min(max(len(push_items), 1), self._PROCESS_THREADS)
max_workers_b=min(max(len(enriched_push_items), 1), self._REQUEST_THREADS)
```
So the max value is `max_workers_a * max_workers_b`

If we change default to proposed and we woudl have jsut one NVR to push
```
    _REQUEST_THREADS = int(os.environ.get("COMMUNITY_PUSH_REQUEST_THREADS", "5"))
    _PROCESS_THREADS = int(os.environ.get("COMMUNITY_PUSH_PROCESS_THREADS", "2"))
```

We would allow just 2 threads to run in parallel. Even though we could manage to upload images to 10 different regions.

Moreover even though we would push 4 nvrs i the pubtask we would still not reach expected 10 threads which we have nowadays with
```
COMMUNITY_PUSH_REQUEST_THREADS = 1 
COMMUNITY_PUSH_PROCESS_THREADS = 10
```

This is reason why nested threadpools does not scale well.

Reply:
@lslebodn I've flattened it by removing the nested thread pool.

PTAL again